packages/calcite-components/.stylelintrc.cjs

```diff
@@ -1,7 +1,13 @@
 // @ts-check
 
 // ⚠️ AUTO-GENERATED CODE - DO NOT EDIT
-const customFunctions = [];
+const customFunctions = [
+  "get-trailing-text-input-padding",
```

Comment:
This code is auto-generated by [this script](https://github.com/Esri/calcite-design-system/blob/dev/support/updateStylelintCustomSassFunctions.ts) and shouldn't be edited directly. IIRC @jcfranco set it up if you need more info.